configure.ac

```diff
@@ -156,6 +156,12 @@ case "$host_cpu" in
 		AH_TEMPLATE(FLAC__CPU_PPC, [define if building for PowerPC])
 		asm_optimisation=$asm_opt
 		;;
+	arm*|aarch64*)
```

Comment:
I believe all we need is:
`arm64|aarch64)`

Otherwise, armv7 gets caught up here too (and is still very common). I don't believe we need anything else here, but someone with more architecture knowledge can chime in.